src/features/pages/home/live-pricing/table-component/live-market-table.tsx

```diff
@@ -0,0 +1,158 @@
+import React, { useCallback, useEffect, useMemo, useRef, useState } from 'react'
+import {
+    flexRender,
+    getCoreRowModel,
+    getSortedRowModel,
+    useReactTable,
+} from '@tanstack/react-table'
+import type { SortingState } from '@tanstack/react-table'
+import { TAvailableLiveMarkets, TMarketData } from '../types'
+import useLiveColumns from '../use-live-columns'
+import { table_row_header, table_row_data } from './live-pricing.module.scss'
+import Flex from 'features/components/atoms/flex-box'
+import useRegion from 'components/hooks/use-region'
+import Typography from 'features/components/atoms/typography'
+import Link from 'features/components/atoms/link'
+import { Localize } from 'components/localization'
+
+export type TLiveMarketTableProps = {
+    selected_market: TAvailableLiveMarkets
+    link_to: string
+    display_name: string
+}
+
+const LiveMarketTable = ({ selected_market, link_to, display_name }: TLiveMarketTableProps) => {
+    const { is_eu } = useRegion()
+    const [initial_loaded, setInitialLoaded] = useState(false)
+
+    const [rawMarketsData, setRawMarketsData] = useState()
```

Comment:
Please correct me if should be `raw`...and if raw, what it means?
```suggestion
    const [row_markets_data, setRowMarketsData] = useState()
```